scripts/lib/types.ts

```diff
@@ -0,0 +1,120 @@
+import * as admin from "firebase-admin";
+
+// re-export type definitions to decrease declaration verbosity in other files
+// and for some reason, ESLint thinks they're unused, thus the disable comments
+//
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export import CollectionReference = admin.firestore.CollectionReference;
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export import DocumentData = admin.firestore.DocumentData;
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export import DocumentReference = admin.firestore.DocumentReference;
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export import DocumentSnapshot = admin.firestore.DocumentSnapshot;
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export import QueryDocumentSnapshot = admin.firestore.QueryDocumentSnapshot;
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export import Timestamp = admin.firestore.Timestamp;
+
+export type LogFunction = typeof console.log;
+
+export type Incomplete<T> = {
+  [K in keyof T]?: T[K] extends object ? Incomplete<T[K]> : T[K];
+};
+
+export type SimulationName = "chat" | "experience" | "seat";
```

Comment:
Currently supported simulations. Right now hardcoded, there isn't much need of dynamic loading and determining for only a handful.